proxy/logging/LogFilter.h

```diff
@@ -417,18 +421,18 @@ wipeField(char **dest, char *field)
             temp_text[i] = 'X';
           }
           temp_text += (p3 - p2);
-          memcpy(temp_text, p3, ((buf_dest + strlen(buf_dest)) - p3));
+          memcpy(temp_text, p3, ((buf_orig_dest + strlen(buf_orig_dest)) - p3));
         } else {
-          for (int i = 0; i < ((buf_dest + strlen(buf_dest)) - p2); i++) {
+          for (int i = 0; i < ((buf_orig_dest + strlen(buf_orig_dest)) - p2); i++) {
             temp_text[i] = 'X';
           }
         }
       } else {
         return;
       }
 
-      tmp_text[strlen(buf_dest)] = '\0';
-      strcpy(*dest, tmp_text);
+      tmp_text[strlen(buf_orig_dest)] = '\0';
+      orig_dest ? strcpy(*orig_dest, tmp_text) : strcpy(*dest, tmp_text);
```

Comment:
Does this mean `orig_dest` will have the wiped value?

Reply:
Yes, that's correct. `orig_dest` will have the original (raw) log field string with the identified part wiped out. `dest` OTOH, is the normalized log field string with the identified paert wiped out, which will cause the log output to be printed as normalized instead of raw string